lib/suspenders/generators/app_generator.rb

```diff
@@ -165,18 +166,22 @@ def setup_git
       end
     end
 
+    def create_local_heroku_setup
+      say "Creating local Heroku setup"
+      build :create_review_apps_setup_script
+      build :create_heroku_application_manifest_file
+      build :create_deploy_script
```

Comment:
We need to move up here (and the related spec) the `set_up_heroku_specific_gems` step.

Thank you!